guides/07_FAQs.md

```diff
@@ -59,6 +59,12 @@ Note that due to limitations in transpilers, requirements of the web platform mi
 
 [Android Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) has a known issue switching between bluetooth audio output devices. While clients can continue the meeting using the default device, there is a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) related to switching to an bluetooth audio output.
```

Comment:
While you're here: s/an bluetooth/a Bluetooth

Reply:
Fixed